app/Http/Controllers/Api/LicensesController.php

```diff
@@ -304,6 +308,122 @@ public function selectlist(Request $request)
 
         return (new SelectlistTransformer)->transformSelectlist($licenses);
     }
+    /**
+     * Checkout a License
+     *
+     * @author [D.Valin] [david.valin@deivishome.com] based on [A. Gianotto] [<snipe@snipe.net>]
+     * @param int $LicenseId
+     * @since [v4.0]
+     * @return JsonResponse
+     * Added functions to checkout license to user and asset
+     */
+    public function checkout(Request $request, $licenseId, $seatId = null)
+    {
+            if (!$license = License::find($licenseId)) {
+                return response()->json(Helper::formatStandardApiResponse('error', null, trans('admin/licenses/message.not_found')));
+           }
+            $this->authorize('checkout', License::class);
+           $licenseSeat = $this->findLicenseSeatToCheckout($license, $seatId);
+           $licenseSeat->user_id = $request->filled('checkout_to');
+
+           $checkoutMethod = 'checkoutTo'.ucwords(request('checkout_to_type'));
+           if ($this->$checkoutMethod($request, $licenseSeat)) {
+                return response()->json(Helper::formatStandardApiResponse('success', null, trans('admin/licenses/message.checkout.success')));
+           }
+           return response()->json(Helper::formatStandardApiResponse('error', null, trans('Something went wrong handling this checkout.')));
```

Comment:
I'm not sure this `trans()` is correct. It's looking for a matching key in the `resources/lang/en` directory, not an actual string

Reply:
Dear Snipe i set trans based on other api responses so if there is a translation of the language it should return the translated chain, if not, i will return the eng language chain, thats what i though.

Reply:
That's not how that works. In later versions of Laravel, you can do that using `__('messages.welcome')`, but we don't use that method currently.